src/lib/linear_rings_are_closed.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Returns a Boolean indicating whether all the linearRings
+ * in an array of them, as in the coordinates of a Polygon,
+ * are closed.
+ *
+ * @param {Array<Array<Array<number>>>} rings
+ * @return {Boolean} `true` if all rings are closed.
+ */
+function linearRingsAreClosed(rings) {
```

Comment:
I think this is just used to throw errors. Might be useful in tests to make sure nothing is screwing this up, but throwing this error isn't problematic as GeoJSON doesn't say this can't happen.